tests/unit/test_consts.py

```diff
@@ -0,0 +1,27 @@
+import itertools
+
+from apify_shared.consts import BOOL_ENV_VARS, DATETIME_ENV_VARS, FLOAT_ENV_VARS, INTEGER_ENV_VARS, STRING_ENV_VARS, ApifyEnvVars
+
+
+class TestConsts:  # noqa: D101
```

Comment:
This `noqa` is not needed (I wonder why `flake8-noqa` didn't report it...)
```suggestion
class TestConsts:
```